packages/core/src/components/MarkdownContent/MarkdownContent.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { makeStyles } from '@material-ui/core';
+import ReactMarkdown from 'react-markdown';
+import gfm from 'remark-gfm';
+import React from 'react';
+import { BackstageTheme } from '@backstage/theme';
+import { CodeSnippet } from '../CodeSnippet';
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  markdown: {
+    '& table': {
+      borderCollapse: 'collapse',
+      border: `1px solid ${theme.palette.border}`,
+    },
+    '& th, & td': {
+      border: `1px solid ${theme.palette.border}`,
+      padding: theme.spacing(1),
+    },
+    '& td': {
+      wordBreak: 'break-word',
+      overflow: 'hidden',
+      verticalAlign: 'middle',
+      lineHeight: '1',
+      margin: 0,
+      padding: theme.spacing(3, 2, 3, 2.5),
+      borderBottom: 0,
+    },
+    '& th': {
+      backgroundColor: theme.palette.background.paper,
+    },
+    '& tr': {
+      backgroundColor: theme.palette.background.paper,
+    },
+    '& tr:nth-child(odd)': {
+      backgroundColor: theme.palette.background.default,
+    },
+
+    '& a': {
+      color: theme.palette.link,
+    },
+    '& img': {
+      maxWidth: '100%',
+    },
+  },
+}));
+
+/**
+ * MarkdownContent. Renders markdown (CommonMark, optionally with [GFM](https://github.com/remarkjs/remark-gfm)) to formatted HTML.
+ */
+type Props = {
+  content: string;
+  enableGfm?: boolean;
+};
+
+const renderers = {
+  code: ({ language, value }: { language: string; value: string }) => {
+    return <CodeSnippet language={language} text={value} />;
+  },
+};
+
+export const MarkdownContent = ({ content, enableGfm = false }: Props) => {
```

Comment:
Is there any reason to not enableGfm by default?

Reply:
One thought of mine was to have more control over when URLs are automatically converted to links. But basically the advantages outweigh the disadvantages. I am indifferent here. Could enable it by default.

Reply:
Yeah, I also don't think it's a big deal either way. Just thinking most people probably assume GFM features as the default.

It looks like react-markdown uses remark 🎉, which I believe would be CommonMark by default. Can't recall if GFM is a 100% compatible superset or if there are some conflicting differences. On a side note, techdocs/mkdocs uses python-markdown which follows neither directly and doesn't behave the same for things like bulleted lists. The joys of competing markdown standards 😞.